src/Simplic.DocumentProcessing/PdfLayer/PdfLayerMode.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Simplic.DocumentProcessing
+{
+    /// <summary>
+    /// Add layer mode
+    /// </summary>
+    public enum PdfLayerMode
+    {
+        /// <summary>
+        /// Add layer to first page
+        /// </summary>
+        FirstPage = 1,
+
+        /// <summary>
+        /// Add layer to all pages
+        /// </summary>
+        EachPage = 2

```

Comment:
im not sure if this name is correct. `AllPages` or `EntireDocument` would be a better fit.